F/FilePathsBase/Deps.toml

```diff
@@ -5,3 +5,7 @@ Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+
+["0.6-0"]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Can probably be test only deps?

Reply:
No, I use it in the main package to provide a default test suite for new path types.

Reply:
Ok, cool. That didn't show up when I searched.

Reply:
https://github.com/rofinn/FilePathsBase.jl/blob/master/src/test.jl